configure.ac

```diff
@@ -27,17 +27,14 @@ AH_TOP([#define LIBSECP256K1_CONFIG_H])
 AH_BOTTOM([#endif /*LIBSECP256K1_CONFIG_H*/])
 AM_INIT_AUTOMAKE([foreign subdir-objects])
 
+AM_PROG_AR # must be called before LT_INIT
```

Comment:
Why the comment does matter? Any docs link?

Reply:
./autogen.sh will warn if you have it the other way around.

The `AM_PROG_AR` felt a little bit out of place here (because all the other tools are detected below), so I thought a comment is good. Another way is to move `AC_PROG_CC` and `AM_PROG_AS` up, maybe I'll do this.

Reply:
Fixed.

Reply:
> ./autogen.sh will warn if you have it the other way around.

`warning: LT_INIT was called before AM_PROG_AR`